test/pseudo-tty/test-tty-color-support.out

```diff
@@ -1,8 +1 @@
 (node:*) Warning: The 'NO_COLOR' env is ignored due to the 'FORCE_COLOR' env being set.
-(node:*) Warning: The 'NO_COLOR' env is ignored due to the 'FORCE_COLOR' env being set.
-(node:*) Warning: The 'NO_COLOR' env is ignored due to the 'FORCE_COLOR' env being set.
-(node:*) Warning: The 'NO_COLOR' env is ignored due to the 'FORCE_COLOR' env being set.
-(node:*) Warning: The 'NODE_DISABLE_COLORS' env is ignored due to the 'FORCE_COLOR' env being set.
-(node:*) Warning: The 'NODE_DISABLE_COLORS' env is ignored due to the 'FORCE_COLOR' env being set.
-(node:*) Warning: The 'NODE_DISABLE_COLORS' env is ignored due to the 'FORCE_COLOR' env being set.
-(node:*) Warning: The 'NODE_DISABLE_COLORS' env is ignored due to the 'FORCE_COLOR' env being set.
```

Comment:
Ideally, it would print both warnings one time each, I would think.

Reply:
Environment variables are normally only passed through once. I do not think it's required to log this warning multiple times.

Reply:
I think that looks at it through our lens and not the end user's lense. The end user may assume that `NO_COLOR` is being ignored but that `NODE_DISABLE_COLORS` should work because no warning is being generated for that.

But I may be overthinking it. It's not a big deal anyway. Just thinking ideally we'd want to list everything we're ignoring rather than just the first thing we ignore.

Reply:
I changed it to include both environment variables if that applies.